### DISCUSSION TITLE
PSRAM crashes on 252MHZ overclock (set in code , not from the IDE menu)

### DISCUSSION THREAD

**Discussion by Macrosii** [earlephilhower/arduino-pico#2619] [Q&A] [answered]
High.  I am using a PICO2 Plus with PSRAM and running the Adafruit "PicoDVI" library.  This library over-clocks the processor in code to 252Mhz to get the PIO outputting at the correct speed for DVI.  I tried adding some PSRAM stuff (fame buffer takes up a lot of the normal RAM) and I am experiencing crashes.  Is the PSRAM access speed set up from the IDE menu ?

**Comment by earlephilhower** [comment, marked as answer]
PSRAM is driven from `clk_sys`, so if you change that then you need to change the divider or it'll go out of spec and the PSRAM chip will probably go very badly.  See:
https://github.com/earlephilhower/arduino-pico/blob/ca30518510c9a5f692e54271552cabef180b2155/cores/rp2040/psram.cpp#L198-L231

**Comment by Macrosii** [comment]
Many Thanks.  That did it, I changed the clock divider and it running ok .

**Comment by FreddyVRetro** [comment]
252MHz is maybe Ok, but above 266MHz, need to change other things (rxdelay to 4 for example)
Is there somewhere a guide for all the frequency ?
Any strong RAM Test code somewhere ?
